ec2api/api/security_group.py

```diff
@@ -163,8 +163,8 @@ def check_and_repair_default_groups(self, os_groups, db_groups):
                 db_group = db_groups_dict.get(os_group)
                 if db_group and db_group == vpc['id']:
                     continue
-            had_to_repair = True
-            _create_default_security_group(self.context, vpc)
+            #had_to_repair = True  #JNT226 loop in describecall
```

Comment:
Instead of commenting out the lines, can we remove this whole method "check_and_repair_default_groups" because it is basically no longer useful?